rs-react-app/src/__tests__/Card.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { Card } from '../components/Card.tsx';
+import type { Character } from '../api/rickAndMorty.ts';
+
+describe('Card', () => {
+  const mockCharacter: Character = {
+    id: 1,
+    name: 'Rick',
+    status: 'Alive',
+    species: 1,
+    image: 'https://ex.com/${id}.png',
```

Comment:
Это строка в кавычках '...', поэтому переменная id не интерполируется. Получится буквально: src="https://ex.com/${id}.png"

```suggestion
image: `https://ex.com/${mockCharacter.id}.png`
```

Reply:
у меня не получилось
я же не могу ссылаться на mockCharacter.id внутри самого mockCharacter -  объект ещё же не завершён